README.md

```diff
@@ -139,6 +139,18 @@ In order to enable source map support you have to specify path to the source map
 
 *Please note* that Airbrake backend downloads source map file in order to process backtrace. This means that source map should be publicly accessible via HTTP. So, for example, don't expect source map support to work on your local webserver running on `localhost`.
 
+You can also specify source map URL in the notice context using `sourceMaps` map, where key is shell file name pattern and value is source map URL:
```

Comment:
``using `sourceMaps` map`` looks a bit confusing in JS. It's technically an object, not a map.

I suggest:

> You can also specify a source map URL in the notice context using the special `sourceMaps` object